src/components/Question/Question.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Prism from 'prismjs';
+
+import '../../assets/css/prism.css';
+
+export default class Question extends Component {
+  componentDidMount() {
+    Prism.highlightAll();
+  }
+
+  render() {
+    const {
+      answerClickHandler,
+      question,
+    } = this.props;
+
+    const {
+      title,
+      description,
+      codeSnippet,
+      codeSnippetLang,
+      answers,
+    } = question;
+
+    const sendAnswer = (answer) => {
+      if (answerClickHandler) {
+        answerClickHandler(answer);
+      }
+    };
+
+    if (!question || !answers) {
+      return null;
+    }
+
+    return (
+      <div>
+        <h1>{title}</h1>
+        {description ? <p>{description}</p> : null}
+
+        {
+          codeSnippet
+            ? (
+              <pre>
+                <code className={`language-${codeSnippetLang}`}>
+                  {`${codeSnippet}`}
+                </code>
+              </pre>
+            )
+            : null
+        }
+
+        <ul>
+          {answers.map((answer) => (
+            <li key={answer.id}>
+              <button type="button" onClick={sendAnswer.bind(this, answer)} onKeyDown={sendAnswer.bind(this, answer)}>{answer.text}</button>
```

Comment:
Is there a reason you need `onKeyDown` listener as well?

Reply:
because of this rule
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md

Reply:
For some reason I thought the click event captured keydown just the way it does touch, but clearly I was wrong.